setup.py

```diff
@@ -24,6 +24,8 @@
         'kubernetes<9.0.0,>=8.0.0',  # to satisfy openshift package requirement
         'openshift',
         'boto3',
+        'munch',
+        'ipdb'
```

Comment:
I don't think we need to have ipdb as part of the installation. If user want to debug it needs to be installed by the user in the venv.

Reply:
Done